pkg/registry/apis/playlist/register.go

```diff
@@ -32,18 +33,21 @@ type PlaylistAPIBuilder struct {
 	namespacer request.NamespaceMapper
 	gv         schema.GroupVersion
 	features   featuremgmt.FeatureToggles
+	kvStore    *kvstore.NamespacedKVStore
 }
 
 func RegisterAPIService(p playlistsvc.Service,
 	apiregistration builder.APIRegistrar,
 	cfg *setting.Cfg,
 	features featuremgmt.FeatureToggles,
+	kvStore kvstore.KVStore,
 ) *PlaylistAPIBuilder {
 	builder := &PlaylistAPIBuilder{
 		service:    p,
 		namespacer: request.GetNamespaceMapper(cfg),
 		gv:         playlist.PlaylistResourceInfo.GroupVersion(),
 		features:   features,
+		kvStore:    kvstore.WithNamespace(kvStore, 0, "storage.dualwriting."+cfg.StackID),
```

Comment:
Do we actually need the stack id here? the kvstore is already specific to the stack.
```suggestion
		kvStore:    kvstore.WithNamespace(kvStore, 0, "storage.dualwriting"),
```